src/OracleService/OracleService.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Neo.ConsoleService" Version="1.0.0" />
+    <PackageReference Include="NeoFS.API" Version="0.0.18" />
```

Comment:
Where is the source of this package?

Reply:
This is a ported version of fsapi go. At present, it is a private library, and the official version will be released after the test.

Reply:
It's better to add the version number as "1.0.0" , formal release version, instead of the "0.0.18" some version like testing.